### PR TITLE
Add Kollaborate Folder Watcher 1.0.0.0

### PR DESCRIPTION
**Pull request by NaanProphet**
Companion application for Digital Rebellion's Kollaborate video editing
servers to automatically watch folders for files and upload them.

### CODE REVIEW

File: Casks/kollaborate-folder-watcher.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'kollaborate-folder-watcher' do
+  version '1.0.0.0'
+  sha256 '31652e3b275a566157da2fcca19740f1451d8bff24a1fdf4ce75c0c69b4ed6b5'
+
+  # digitalrebellion.com is the official download host per the vendor homepage
+  url "http://www.digitalrebellion.com/download/kollabfolderwatcher?version=#{version.no_dots}"
+  name 'Kollaborate Folder Watcher'
+  homepage 'https://www.kollaborate.tv/resources'
+  license :commercial
```
Comment (jawshooah) [inline Casks/kollaborate-folder-watcher.rb]: It's listed as free on the homepage. Why `:commercial`?

**Comment by SirDap** [comment]
Same proposed reasoning as https://github.com/caskroom/homebrew-cask/pull/17158#issuecomment-172290925

**Comment by SirDap** [comment]
Updated license, thanks!

**Comment by adidalal** [comment]
Thank you for the contribution. It was merged directly as f986f4718aeeb30ae583d1138b5aa7846f3c144b to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**